plutus-core/untyped-plutus-core/testlib/Scoping/Spec.hs

```diff
@@ -78,6 +79,11 @@ test_names = testGroup "names"
         T.BindingRemovalOk  -- COKC removes branches, which may (and likely do) contain bindings.
         T.PrerenameYes
         (evalSimplifierT . caseReduce)
+    , T.test_scopingGood "case-apply"
+        (genTerm @DefaultFun)
+        T.BindingRemovalOk
```

Comment:
I think it's `BindingRemovalNotOk`? Not sure. Does anything break if you make it that?